pkg/cmd/roachtest/tests/rapid_restart.go

```diff
@@ -51,8 +51,8 @@ func runRapidRestart(ctx context.Context, t test.Test, c cluster.Cluster) {
 			go func() {
 				err := c.RunE(ctx, nodes,
 					`mkdir -p {log-dir} && ./cockroach start-single-node --insecure --store={store-dir} `+
-						`--log-dir={log-dir} --cache=10% --max-sql-memory=10% `+
-						`--listen-addr=:{pgport:1} --http-port=$[{pgport:1}+1] `+
+						`--log={log-dir} --cache=10% --max-sql-memory=10% `+
```

Comment:
that `--log` flag is not going to work. I think `--log-dir` is still what you want here, unless you can use `--log='file-defaults: dir: {log-dir}'`

and then separately is this sufficient to make the test pass?

Reply:
> that --log flag is not going to work. I think --log-dir is still what you want here, unless you can use --log='file-defaults: > dir: {log-dir}'

Got it.

> and then separately is this sufficient to make the test pass?

Yes. The test passes with these changes, as far as I can tell.